### PR TITLE
Docs on how to build bzzz

### LINKED ISSUE #187: Documentation to build the drone
### Describe the issue

Add documentation for how to to build the drone from scratch 

* [x] Parts list 
* [x] how to order PCB's
* [x] Link to 3D printed parts where necessary
* [x] Step by step guide to assemble 
* [ ] Video?

### Affected File

_No response_

### Affected Line

_No response_

### PR DESCRIPTION
**Pull request by jamie-54**
## Main Changes

Full documentation on how to build bzzz including  Parts list, how to order PCB's and step by step guide with photos

## Associated Issues

- Closes #187

## Tests

N/A

### CODE REVIEW

**Comment by jamie-54** [comment]
@alphaville @pdavid747 @Runway27 @Yuanbwcx @ejb-11 @Minanchi, This is nearly finished so if you could have a quick look over it soon to see if I have forgotten anything that would be great

**Comment by pdavid747** [comment]
> @alphaville @pdavid747 @Runway27 @Yuanbwcx @ejb-11 @Minanchi, This is nearly finished so if you could have a quick look over it soon to see if I have forgotten anything that would be great

how do I view the md file?

**Review by alphaville — COMMENTED** [review]
@jamie-54 So far, everything looks great. There are just some missing photos towards the bottom of the file.

File: docs/HOW_TO_BUILD_BZZZ.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+## Tools
+You will need the following tolls to build bzzz.
+* Soldering iron with solder
+* Crimp tool
+* 1.5, 2, 2.5 Allen/Hex keys
+* 4mm flat and Phillips 2 screwdrivers
+* side cutting pliers
+* Pliers
+* X, Y, Z Spanners
```
Comment (alphaville) [inline docs/HOW_TO_BUILD_BZZZ.md]: X, Y, Z?

**Comment by jamie-54** [comment]
> @jamie-54 So far, everything looks great. There are just some missing photos towards the bottom of the file.

Think I've added all photos except one with propellers and lid

File: docs/HOW_TO_BUILD_BZZZ.md (hunk 1 above)
Comment (jamie-54) [reply docs/HOW_TO_BUILD_BZZZ.md]: Need to check size of spanners we use/need

**Comment by jamie-54** [comment]
This is read for review except I would like to add the plain controller case lid to the parts list so if #232 could be reviewed and merged first that would be great then I'll add that file thanks

**Comment by jamie-54** [comment]
@alphaville @pdavid747 @Runway27 @Yuanbwcx @ejb-11 @Minanchi, 
This is now fully complete and ready for review

**Review by alphaville — CHANGES_REQUESTED** [review]
The main `README` file needs to be updated: we need a link to "how to build bzzz". Also the link to `SETUP.md` needs to be updated.

File: docs/HOW_TO_BUILD_BZZZ.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
+# How to Build bzzz
```
Comment (alphaville) [inline docs/HOW_TO_BUILD_BZZZ.md]: build

File: docs/HOW_TO_BUILD_BZZZ.md [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+This documentation includes the relevant information on how to build our quadcopter bzzz from scratch.
+
+## Tools
+You will need the following tools to build bzzz.
+* Soldering iron with solder
+* Crimp tool
```
Comment (alphaville) [inline docs/HOW_TO_BUILD_BZZZ.md]: Crimping?

File: docs/HOW_TO_BUILD_BZZZ.md [hunk 4]
```diff
@@ -0,0 +1,421 @@
...
+
+<img width="400" src="images/Receiver_in_case.png">
+  
+* Attach the 14 Way FFC cable to the Pi PCB.
+* Use double sided tape to stick the flight controller (Pi and ESP) to the anti-vibration plate and place the FFC cable through the slot in the controller case.
+* Attach the 14 Way FFC cable to the Sensor PCB.
+
+<img width="400" src="images/FFC.png">
+
+* Attach the reviver wires to the correct header on the Pi PCB.
```
Comment (alphaville) [inline docs/HOW_TO_BUILD_BZZZ.md]: receiver

File: docs/HOW_TO_BUILD_BZZZ.md [hunk 5]
```diff
@@ -0,0 +1,421 @@
...
+* Feed the ESC white signal wires and the single black ground wire through the holes in the controller case.
+* Place the terminal contacts from each ESC wires in the correct position of a Molex KK 5 x 1 Connector Housing (Ground, front right, front left, back right back left).
+* Attach the ESC connector from the previous step to the correct header on the Pi PCB.
+* Feed the wires on output 1 of the UBEC through the holes in the controller case.
+* Place the terminal contacts from output 1 of the UBEC in the correct position of a Molex KK 6 x 1 Connector Housing (Power, 4 blanks, Ground).
+* Attach the Power (UBEC) connector from the previous step to the correct header on the Pi PCB.
+
+<img width="400" src="images/FC_in_case.png">
+
+* If you haven't already, using the USB memory stick follow the [software setup](https://github.com/QUB-ASL/bzzz/blob/main/docs/SETUP.md).
```
Comment (alphaville) [inline docs/HOW_TO_BUILD_BZZZ.md]: update link to `SETUP.md`

**Comment by jamie-54** [comment]
> The main `README` file needs to be updated: we need a link to "how to build bzzz". Also the link to `SETUP.md` needs to be updated.

done

**Review by alphaville — APPROVED** [review]

**Pull request merged**